docs/adding-translations.md

```diff
@@ -0,0 +1,39 @@
+# Adding translations
+
+There are translatable strings from both JS an Python.
+
+We use [Transifex][transifex] for translations on the project.
+First, you have to set a [~/.transifexrc][transifexrc].
+
+!!! warning
+    We only translate strings from the `master` branch of the repository, do not push languages on any other branch because merging translations is incredibly painful.
+
+## Existing languages
+
+
+The command `inv i18n` retrieve all these strings locally that you can then push to Transifex with the command `tx push -s`.
+
+Then go to Transifex and translate new strings.
+
+Once you did this, you can pull these strings with `tx pull` and then compile the retrieved files with `inv i18nc`.
+
+Restart your server and new strings should be translated (don't forget to activate the right language!).
+
+
+## Proposing a new language
+
+Propose the new language on [Transifex][transifex], once accepted you can:
```

Comment:
Actually you can only reach it once you're logged in that's why I redirected to the home. I'm not sure how to describe that. Or maybe with two steps?